docker/docker-compose-3/api.py

```diff
@@ -0,0 +1,74 @@
+import mysql.connector
+from mysql.connector import Error
+from flask import Flask
+import os
+from dotenv import load_dotenv
+from flask import request, jsonify
+
+app = Flask(__name__)
+
+load_dotenv()
+
+def connect():
+    try:
+        connection = mysql.connector.connect(host=os.getenv("MYSQL_HOST"),
+                                            database=os.getenv("MYSQL_DB"),
+                                            user=os.getenv("MYSQL_USER"),
+                                            password=os.getenv("MYSQL_PASSWORD"),
+                                            port=3306)
+        if connection.is_connected():
+            return connection
+
+    except Error as error:
+        print("Error while connecting to MySQL", error)
+
+def add_db():
+    connection = connect()
+    with connection.cursor(dictionary=True) as cursor:
+        cursor.execute(f"SHOW TABLES LIKE 'users'")
+        is_table_exists = cursor.fetchone()
+
+        if not is_table_exists:
+            cursor.execute(f"CREATE TABLE users (id INT AUTO_INCREMENT PRIMARY KEY, name VARCHAR(255))")
+
+@app.route("/<table>", methods = ["POST"])
+def add(table: str):
+    connection = connect()
```

Comment:
it's better to connect ot the DB once in the beggining of the app and not per request